src/BrandwatchReactAuth.js

```diff
@@ -69,14 +76,16 @@ export default class BrandwatchReactAuth extends Component {
 
 BrandwatchReactAuth.propTypes = {
   audience: PropTypes.string.isRequired,
-  domain: PropTypes.string.isRequired,
   backupDomain: PropTypes.string,
   backupRedirect: PropTypes.string,
+  children: PropTypes.node,
+  domain: PropTypes.string.isRequired,
   onCreateStore: PropTypes.func,
 };
 
 BrandwatchReactAuth.childContextTypes = {
   brandwatchAuthGetProfile: PropTypes.func.isRequired,
   brandwatchAuthGetToken: PropTypes.func.isRequired,
+  brandwatchAuthHandleError: PropTypes.func.isRequired,
```

Comment:
I still wonder if this name still describes what it's doing 🤔 It's more of `brandwatchAuthRedirect`?

Reply:
I see what you mean, or maybe even `brandwatchAuthExitApplication`